projects/js-themes-toolkit/packages/liferay-theme-tasks/theme/__tests__/index.test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const {r2} = require('liferay-theme-tasks');
+
+describe('index.js', () => {
+	it('exposes the r2 API from the "main" package resource', () => {
+		expect(require('liferay-theme-tasks')).toBe(require('..'));
```

Comment:
Just proving that `'../index.js'` really _is_ the `"main"` export from the package.